utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.firebase
+
+import java.lang.Exception
+
+/**
+ * Interface that enables exception logging.
+ */
+interface CrashLogger {
+
+  /** @param [exception] : the non fatal exception that is mostly caught in try-catch blocks.
+   * This function basically facilitates custom exception logging to Crashlytics.
```

Comment:
Similar to other comments--Crashlytics isn't relevant to this component from a user's perspective. Users just want to log exceptions. You can note like I suggested in an earlier comment that exceptions may be logged to remote services or locally on disk, but _how_ it does that should be left up to the implementation (consider that the fake can't actually satisfy this requirement since it doesn't use Crashlytics).